test/boost/index_with_paging_test.cc

```diff
@@ -22,14 +22,15 @@ SEASTAR_TEST_CASE(test_index_with_paging) {
         // There should be enough rows to use multiple pages
         auto prepared_id = e.prepare("INSERT INTO tab (pk, ck, v, v2, v3) VALUES (?, ?, 1, ?, ?)").get0();
         auto big_string_v = cql3::raw_value::make_value(serialized(big_string));
-        for (int i = 0; i < 64 * 1024; ++i) {
-            e.execute_prepared(prepared_id, {
+        max_concurrent_for_each(boost::irange(0, 64 * 1024), 50, [&] (auto i) {
```

Comment:
I thought again why this patch brings any performance improvement, and my new guess is that it's because the test is run on two shards, and the concurrency allows sending some additional requests to the second shard while this shard is waiting for a memtable flush. If my guess is correct then this 50 concurrency seems a very crude tool to do that. But I can't argue with the results, so I guess it's better than nothing.

Reply:
Running on >1 shard anyway is a good practice, I guess - more realistic.

Reply:
Yes just please note that this patch doesn't run the loop on more than one shard - the loop still happens on just one shard. Yes, it sends half of the writes to the other shard, but this was also the case before this change. So it's not completely obvious why this change helps. I have some guesses but none of them are very convincing.

Reply:
Changed to 2 https://github.com/scylladb/scylladb/commit/d4697ed21ebfacf55b2423ffa9483d73cf083483